tests/helpers.py

```diff
@@ -397,6 +397,31 @@ def reference_id_from_url(url):
     }
 ]
 
+TEST_SCENE_CATEGORY_ANNOTATIONS = [
+    {
+        "label": f"[Pytest] Scene Category Label ${i}",
+        "reference_id": reference_id_from_url(TEST_IMG_URLS[i]),
+        "taxonomy_name": "[Pytest] Scene Category Taxonomy 1",
+    }
+    for i in range(len(TEST_IMG_URLS))
+]
+
+TEST_DEFAULT_SCENE_CATEGORY_ANNOTATIONS = [
+    {
+        "label": f"[Pytest] Scene Category Label ${i}",
+        "reference_id": reference_id_from_url(TEST_IMG_URLS[i]),
+    }
+    for i in range(len(TEST_IMG_URLS))
+]
+
+TEST_NONEXISTENT_TAXONOMY_SCENE_CATEGORY_ANNOTATIONS = [
+    {
+        "label": "[Pytest] Scene Category Label 0",
+        "reference_id": reference_id_from_url(TEST_IMG_URLS[0]),
+        "taxonomy_name": "[Pytest] Scene Category Taxonomy Nonexistent",
+    }
+]
```

Comment:
Is this going to be re-used? If not I'd define this in the specific test.

Reply:
I think it's okay to leave it, since it's mirrored by `TEST_NONEXISTENT_TAXONOMY_SCENE_CATEGORY_PREDICTIONS` and doesn't clutter up the actual test file. Helpers files, in contrast, may be longer